### DISCUSSION TITLE
submit ERROR: "No such file or directory: ./data/sdwpf_baidukddcup2022_full_full.csv"

### DISCUSSION THREAD

**Discussion by yuhao-morning** [PaddlePaddle/PaddleSpatial#99] [BaiduKDDCup2022] [answered]
My submiited folder structure is :

predict.py
prepare.py
some other .py files

data/sdwpf_baidukddcup2022_full_full.csv
data/sdwpf_baidukddcup2022_full_full_processed_withna.csv
data/config_kdd2022.json

models/trained_config.json
modes/ts_view_modes/***.ckpt

Is there any wrong?

**Comment by ramber1836** [comment]
Have you ever tested your model in your local environment? 
The structure of your local environment should follow the code structure of 'PaddleSpatial/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation/paddlepaddle'.

**Comment by yuhao-morning** [comment]
I tested locally，with my own main program  evaluation.py and data it runs very well. I think it is the directory problem， it cannot find the file. Maybe can you check the log again?



---- Replied Message ----
| From | Xinjiang ***@***.***> |
| Date | 05/24/2022 18:05 |
| To | ***@***.***> |
| Cc | ***@***.******@***.***> |
| Subject | Re: [PaddlePaddle/PaddleSpatial] submit ERROR: "No such file or directory: ./data/sdwpf_baidukddcup2022_full_full.csv" (Discussion #99) |

Have you ever tested your model in your local environment?
The structure of your local environment should follow the code structure of 'PaddleSpatial/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation/paddlepaddle'.

**Comment by ramber1836** [comment, marked as answer]
The evaluation script adopted for the online test is not your version.

Regards,
Xinjiang Lu
